spec/integration/test.js

```diff
@@ -192,6 +194,47 @@ describer('notification api with a live service', function () {
       });
     });
 
+    it('get a letter pdf', (done) => {
+      should.exist(letterNotificationId)
+      pdf_contents = fs.readFileSync('./spec/integration/test_files/one_page_pdf.pdf');
+      var count = 0
+      // make sure test is closed
+      const endTest = (err) => {
+        if (err !== undefined) {
+          done(err);
+        } else {
+          done();
+        }
+      };
```

Comment:
I don't think endTest gives us anything - we never call it from a place where `err` could be undefined. I think this can be deleted and replaced with just calls to `done`